symfony/plugins/orangehrmAttendancePlugin/Api/EmployeeAttendanceSummaryAPI.php

```diff
@@ -106,7 +106,11 @@ public function getValidationRuleForGetAll(): ParamRuleCollection
                 new ParamRule(
                     CommonParams::PARAMETER_EMP_NUMBER,
                     new Rule(Rules::POSITIVE),
-                    new Rule(EmployeeDataGroupReadPermissionRule::class, ['apiv2_attendance_employee_attendance_summary']),
+                    new Rule(Rules::ENTITY_ID_EXISTS, [Employee::class]),
```

Comment:
If we pass un available employee number, we didnt get invalid param exception, thats why i added new Rule(Rules::ENTITY_ID_EXISTS, [Employee::class]).